lib/crypto.js

```diff
@@ -599,17 +599,19 @@ function Certificate() {
 }
 
 
-Certificate.prototype.verifySpkac = function(object) {
+Certificate.prototype.verifySpkac = function verifySpkac(object) {
   return binding.certVerifySpkac(object);
 };
 
 
-Certificate.prototype.exportPublicKey = function(object, encoding) {
+Certificate.prototype.exportPublicKey = function exportPublicKey(object, 
+        encoding) {
```

Comment:
Typically we like to keep the parameters all lined up if on multiple lines. However in the case that a lined up parameter on a new line would exceed the 80 character mark, it would probably be better to just move everything after the `=` to the next line and indent the entire function definition by 2 spaces.

Ditto for the changes below.